### ISSUE TITLE
Accessibility: WAI-ARIA image is missing alternative text

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#3074]
I think this is a non-issue because the font awesome images are handled in some other way, but just wanted to record it just in case. 

Non-text Content
The WAI-ARIA image, created with role="img", does not have an accessible name.

If the image is informative or functional, use either the WAI-ARIA attribute 'aria-label' or 'aria-labelledby' to create an alternative text for the image.

If the image is purely decorative, use the WAI-ARIA attribute role="presentation" instead of role="img" for the image. This will cause assistive technologies to ignore the image.
Use one of the following techniques from WCAG 2:

ARIA6 Using aria-label to provide labels for objects
ARIA10 Using aria-labelledby to provide a text alternative for non-text content

**Comment by michelleif** [comment]
@astridu is this a Sinopia-side thing or a font-awesome responsibility?

**Comment by michelleif** [comment]
@jcoyne please advise what work is needed here?

**Comment by justinlittman** [comment]
Closing as non-actionable.
